### DISCUSSION TITLE
Convert "usage question" issues to discussions

### DISCUSSION THREAD

**Discussion by andersy005** [pydata/xarray#6146] [General]
@pydata/xarray 

The number of open issues has been growing steadily [^1]. Some of the issues are purely [usage questions](https://github.com/pydata/xarray/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22usage+question%22). **In an effort to reduce the issue backlog**, I was wondering if we could convert these usage questions to discussions instead?

![Screen Shot 2022-01-08 at 9 18 52 PM](https://user-images.githubusercontent.com/13301940/148669261-7b4980b3-aea1-4a1d-be87-64e34a4ed201.png)

[^1]: [Dashboard](https://xarray.vercel.app/dashboard)

**Comment by shoyer** [comment]
Sounds good to me for clear usage question. But discussions is also a limited format, really best for Q&A. So if someone has a true discussion topic, I'm also OK with an issue.
